### PR TITLE
Icon on apps settings

### LINKED ISSUE #3987: Add icons to navigation of App Store items
With #3151 we introduced these nice little icons for navigation items on personal & admin settings pages.

capture du 2017-01-19 03-10-45 bildschirmfoto 2017-03-22 um 13 36 19

@jancborchardt @nextcloud/designers

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #3987

### CODE REVIEW

**Comment by skjnldsv** [comment]
I tried to use most of stuff we already use on nextcloud to avoid confusion.
I created/edited some others. Please suggest a modification/new one if you don't like those.
There are some icons (like the key) which I don't like on my pr, I would be glad to see you suggest something else @nextcloud/designers :)

Thanks!
capture d ecran_2017-09-24_08-47-04

File: settings/js/apps.js [hunk 1]
```diff
@@ -40,10 +40,10 @@ OC.Settings.Apps = OC.Settings.Apps || {
 			this._loadCategoriesCall.abort();
 		}
 
-		var categories = [
-			{displayName: t('settings', 'Enabled apps'), ident: 'enabled', id: '0'},
-			{displayName: t('settings', 'Disabled apps'), ident: 'disabled', id: '1'},
-			{displayName: t('settings', 'Your apps'), ident: 'installed', id: '2'}
+		var categories = [,
```
Comment (ChristophWurst) [inline settings/js/apps.js]: remove the `,` please

File: settings/js/apps.js [hunk 2]
```diff
@@ -40,10 +40,10 @@ OC.Settings.Apps = OC.Settings.Apps || {
...
+			{displayName: t('settings', 'Your apps'), ident: 'installed', id: '0'},
+			{displayName: t('settings', 'Enabled apps'), ident: 'enabled', id: '1',},
```
Comment (ChristophWurst) [inline settings/js/apps.js]: remove the `,` please

**Review by ChristophWurst — DISMISSED** [review]

File: settings/js/apps.js (hunk 2 above)
Comment (skjnldsv) [reply settings/js/apps.js]: How did I missed that... 🙈 
Thanks!!

**Review by MorrisJobke — APPROVED** [review]
Awesome 🚀 I like it

bildschirmfoto 2017-09-25 um 15 50 06

**Review by nickvergessen — APPROVED** [review]

**Comment by jancborchardt** [comment]
Awesome stuff @skjnldsv I like it a lot! :)

The key is good too! Maybe only mirror it, so that the key ring is on the top right instead – that’s how keys are most often visualized.

**Comment by skjnldsv** [comment]
@jancborchardt damn, few seconds too late! 😕 
Feel free to provide a pr for that if you really want! 😉

**Pull request merged**